<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Excel to Google Tasks v1.1 by vuongcris4</string>
  </property>
  <widget class="QPushButton" name="updateTasks">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>130</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cập nhật việc cần làm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addTasks">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Thêm việc cần làm</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>271</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Danh sách việc cần làm</string>
   </property>
   <widget class="QListWidget" name="listTasks">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>251</width>
      <height>161</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="reloadBtn">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>-1</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Làm mới</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>224</y>
     <width>152</width>
     <height>11</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>From Trần Duy Vương with love &lt;3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="hdsdBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>238</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Hướng dẫn sử dụng</string>
   </property>
  </widget>
  <widget class="QPushButton" name="logOutBtn">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>241</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Đăng xuất</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
